src/main/java/fr/insalyon/creatis/gasw/script/ExecutionGenerator.java

```diff
@@ -182,11 +187,22 @@ public String loadInputs(String serviceCall, List<URI> downloads) throws Excepti
         return velocity.merge().toString();
     }
 
+    public String loadInputs(String executableName, String invocationString) throws Exception {
```

Comment:
Is it used somewhere ?

Reply:
No, it was implemented before using the global script, its no longer required.